kafka/conn.py

```diff
@@ -167,9 +182,13 @@ class BrokerConnection(object):
         'metric_group_prefix': '',
         'sasl_mechanism': 'PLAIN',
         'sasl_plain_username': None,
-        'sasl_plain_password': None
+        'sasl_plain_password': None,
+        'sasl_servicename_kafka':'kafka'
```

Comment:
Yes, this is exactly the same as sasl.kerberos.service.name in the java client, i'll change the name accordingly